src/parser/parser.yy

```diff
@@ -614,6 +617,9 @@ alter_schema_prop_list
 
 alter_schema_prop_item
     : KW_TTL_DURATION ASSIGN INTEGER {
+        if ( $3 < 0) {
```

Comment:
Extra space.

Reply:
Good catch

Reply:
Thank you for your careful guidance. @dangleptr